installscripts/jazz-terraform-unix-noinstances/provisioners.tf

```diff
@@ -22,38 +22,49 @@ resource "null_resource" "configureExistingJenkinsServer" {
   provisioner "local-exec" {
     command = "${var.configurescmelb_cmd} ${var.scmbb} ${lookup(var.scmmap, "elb")} ${var.jenkinsattribsfile} ${var.jenkinsjsonpropsfile} ${var.scmclient_cmd}"
   }
+
+  #Because we have to provision a preexisting machine here and can't use the terraform ses command,
+  #we must use sed to insert AWS creds from the provisioner environment into a script chef will run later, before we copy the cookbook to the remote box.
+  provisioner "local-exec" {
+    command = "sed -i 's/AWS_ACCESS_KEY=.*.$/AWS_ACCESS_KEY=$AWS_ACCESS_KEY_ID/g' ${var.cookbooksDir}/jenkins/files/credentials/aws.sh"
```

Comment:
This line which actually replace the line `AWS_ACCESS_KEY="Replaced with sed command from create.sh"` with `AWS_ACCESS_KEY=$AWS_ACCESS_KEY_ID`. We will need the value of AWS_ACCESS_KEY_ID instead of `$AWS_ACCESS_KEY_ID`. 

This is causing all the jenkins builds to fails with the error:
`ERROR: No such property: AWS_ACCESS_KEY_ID for class: groovy.lang.Binding`

Reply:
Will fix. I don't have any visibility into to the CI builds so it's hard for me to know when stuff has been broken.

Reply:
I understand it is hard to troubleshoot issues when you have no visibility to the CI builds. We usually try to share this info on our slack channel once we determine what changes broke the build.

Reply:
Fixed in #137